src/compile.rs

```diff
@@ -633,6 +631,22 @@ fn derive_functions<'a>(
                     }
                 }
             }
+            if let AnnotationInfo::NoDerive = annotation {
+                saw_no_derive = true;
+            }
+        }
+        if saw_derive && saw_no_derive {
+            // TODO: Add error
```

Comment:
Is this a TODO we want to do now?  Is it something more complicated than just saying we saw a `@derive` and a `@noderive` for type X?